src/Sentry/Internal/Hub.cs

```diff
@@ -496,4 +509,10 @@ public SentryId LastEventId
             return currentScope.Key.LastEventId;
         }
     }
+
+    bool IHubEx.IsSentryRequest(Uri requestUri)
```

Comment:
With explicit interface implementation it requires the type to be casted to `IHubEx` so this is an odd use of it here on an internal interface